inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/MqttReader.java

```diff
@@ -116,8 +119,8 @@ private void connect() {
 
                     @Override
                     public void connectionLost(Throwable cause) {
-                        LOGGER.info("the mqtt connection is lost, try to reconnect. jobId:{},serverURI:{},clientId:{}",
-                                instanceId, serverURI, clientId);
+                        LOGGER.error("the mqtt jobId:{}, serverURI:{}, connection lost, {} ", instanceId,
+                                serverURI, cause.getMessage());
```

Comment:
Do we need to print all stacks of the exception? Maybe using `cause` instead of `cause.getMessage()` is better if required.